### PR TITLE
WIP: Issue 124

### PR DESCRIPTION
**Pull request by selketjah**
Concerning: #124 

Added:
- supported colors
image

- custom color
image

- NoColor
image

- sizes
image

### CODE REVIEW

**Comment by MangelMaxime** [comment]
It's looking great :)

A quick note, I don't think Fulma support `text custom color` yet

**Comment by selketjah** [comment]
I mean "the text about custom color" 
image

(as you can see, the example isn't working yet)

File: docs/scss/main.scss [hunk 1]
```diff
@@ -27,6 +27,7 @@ $primary: $dark-blue;
 @import "./callout";
 @import "./prism";
 @import "./../css/prism.min.css";
+@import "customcolor";
```
Comment (MangelMaxime) [inline docs/scss/main.scss]: You need to include this file before Bulma import.

File: docs/scss/customcolor.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import "./../../node_modules/bulma/sass/utilities/initial-variables";
```
Comment (MangelMaxime) [inline docs/scss/customcolor.scss]: This include is already added in `main.scss`

**Comment by selketjah** [comment]
Ready to merge

**Comment by MangelMaxime** [comment]
Awesome thank you :)

**Pull request merged**